packages/amplify-graphql-auth-transformer/src/graphql-auth-transformer.ts

```diff
@@ -334,8 +340,39 @@ export class AuthTransformer extends TransformerAuthBase implements TransformerA
     this.authModelConfig.forEach((acm, modelName) => {
       const def = context.output.getObject(modelName)!;
       const modelHasSearchable = def.directives.some((dir) => dir.name.value === 'searchable');
+      const ownerFields = getOwnerFields(acm);
```

Comment:
How does it work with SQL datasource? We do not auto generate any of these implicit fields for SQL.